### ISSUE TITLE
strings solution is incorrect

### ISSUE THREAD

**Issue by rshpeley** [workshopper/javascripting#295]
The solution of Exercise 3 is incorrect. It gives:
``` javascript
console.log('this is a string')
```
whereas the correct solution should be:
``` javascript
const someString = 'this is a string'
console.log(someString)
```
1. start `javascripting`
2. select strings
3. read what is required (using the `someString` constant variable)
4. create the `strings.js` file with the correct solution
5. verify with `javascripting verify strings.js`
6. the output taken from `solutions/strings/index.js` is the incorrect solution

Since the lesson asks for `someString` to be created and used, the solution should also reflect the same requirement. 

The fix is to modify the contents of `solutions/strings/index.js` to the correct solution.

This issue is related to issues:
#176 marked as fixed for a different problem but perhaps led to this one
#196 because only the output is tested and not the source code.

**Comment by ledsun** [comment]
I agree with your point.
Creating variables in problems and not using variables in solutions is confusing.

I fixed it at https://github.com/workshopper/javascripting/commit/810f4811c81c634c8178b7e1543d4ac505b15531.

@sethvincent Could you publish 2.7.4 ?

**Comment by ccarruitero** [comment]
@ledsun I think we should prepare the repository with the version change first, (maybe a changelog too?) and then ping someone for publish.

**Comment by ledsun** [comment]
This is fixed at 2.7.4
